README.md

```diff
@@ -1,9 +1,199 @@
+
+| Time | Room 1 | Room 2 |
+|:------------- |:-----:|:-----:|
+| 1 - 2 PM | Unix shell 1 (Valentina) | Best practices in scientific computing (Chris) |
+| 2 - 3 PM | Unix shell 1 (Valentina) | Best practices in scientific computing (Chris) |
+| 3 - 4 PM | Git/GitHub 1 (Bernease) | Web development (Anisha) |
+| 4 - 5 PM | Git/GitHub 2 (Bernease) | Jupyter/Interactive computing (Dylan) |
+| 5 - 6 PM | Python programming 1 (Tal) | Intro to R (Jeanette) |
+| 6 - 7 PM | Python programming 2 (Tal) | Intro to R (Jeanette) |
+
+
+
+### Tuesday (9/5)
+
+8 AM - 9 AM: Breakfast (LH)
+
+9 AM - 12 PM: Cloud computing (Tara Madhyastha; OUGL)
+
+12 PM - 1 PM: Lunch (LH)
+
+1 PM - 3 PM: Team project pitches (PAB5)
+
+3 PM - 6 PM: Team projects (DSS)
+* Afternoon breakout sessions (optional):
+    * 3 PM: AIBS data-sets (Nick Cain)
+    * 4 PM: Advanced data manipulation in pandas (Tal Yarkoni)
+
+6 PM - 12 AM: Dinner (on your own) and hackathon sessions (LH)
+
+
+### Wednesday (9/6)
+
+8 AM - 9 AM: Breakfast (LH)
+
+9 AM - 11 AM: Machine learning (Jake Vanderplas; OUGL)
+
+11 AM - 12 PM: Machine learning in neuroimaging (Chris Holdgraf; OUGL)
+
+12 PM - 1 PM: Lunch (LH)
+
+1 PM - 6 PM: Team projects (DSS)
+* Afternoon breakout sessions (optional):
+    * 3 PM: Statistical practice in neuroimaging (JB Poline)
+    * 4 PM: Power in fMRI (Jeanette Mumford)
+
+6 PM - 12 AM: Dinner (on your own) and hackathon sessions (LH)
+
+
+### Thursday (9/7
+
+8 AM - 9 AM: Breakfast (LH)
+
+9 AM - 12 PM: Neuroimaging pipelines (Satra Ghosh; OUGL)
+
+12 PM - 1 PM: Lunch (LH)
+
+3 - 6 PM: Team projects (DSS)
+* Afternoon breakout sessions (optional):
+    * 3 PM: Advanced time-series analysis (Bing Brunton)
+
+6 PM - 12 AM: Dinner (on your own) and hackathon sessions (LH)
+
+
+### Friday (9/8)
+
+8 AM - 9 AM: Breakfast (LH)
+
+9 AM - 12 PM: Team projects (DSS)
+
+12 PM - 1 PM: Lunch (LH)
+
+1 PM - 2 PM: Team projects (DSS)
+
+2 PM - 5 PM: Project presentations (DSS)
+
+5 PM - 6 PM: Summary and feedback (PAB5)
+
+6:30 PM: Farewell reception and Lake Union Cruise
```

Comment:
We have to be at the marina at 6:00 for boarding, and sail at 6:30. 

We have to get there using public transit (which is easy!): bus line 70 picks up on 15th Ave and Campus Parkway, and drops off at SLU, a short walk from the AGC marina, where we will board. However, we need to shift the schedule a bit to give us time for transit.